src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -729,11 +829,70 @@ public int compareTo(BridgeState o)
             boolean otherOperational = o.isOperational();
 
             if (meOperational && !otherOperational)
+            {
                 return -1;
+            }
             else if (!meOperational && otherOperational)
+            {
                 return 1;
+            }
+
+            return this.getEstimatedVideoStreamCount()
+                - o.getEstimatedVideoStreamCount();
+        }
+
+        private int getEstimatedVideoStreamCount()
+        {
+            int videoStreamEstimate
+                = videoStreamCount != null ? videoStreamCount : 0;
+
+            if (videoStreamCountDiff != null)
```

Comment:
That should do it; Alternatively, you could make the `videoStreamCountDiff` a primitive and sum it unconditionally (null is the same a 0). I don't have any preference tho :)